tools/StaticAnalysis/Exceptions/Az.Resources/BreakingChangeIssues.csv

```diff
@@ -109,4 +109,12 @@
 "Microsoft.Azure.PowerShell.Cmdlets.ResourceManager.Cmdlets.dll","Microsoft.Azure.Commands.ResourceManager.Cmdlets.Implementation.SetAzurePolicyAssignmentCmdlet","Set-AzPolicyAssignment","0","1050","The parameter set 'NameParameterSet' for cmdlet 'Set-AzPolicyAssignment' has been removed.","Add parameter set 'NameParameterSet' back to cmdlet 'Set-AzPolicyAssignment'."
```

Comment:
@Tiano2017 when is it planned to publish version 2.3.0-preview of `Micorosft.Azure.Management.ResourceManager` to NuGet?

Reply:
we'll publish it in the next few days. Actually the new release is going to be 2.4.0-preview.